packages/slate-editor/src/extensions/tables/TablesExtension.tsx

```diff
@@ -31,6 +33,19 @@ export function TablesExtension(): Extension {
     return {
         id: EXTENSION_ID,
         normalizeNode: [normalizeTableAttributes, normalizeRowAttributes, normalizeCellAttributes],
+        deserialize: {
+            element: composeElementDeserializer({
+                TABLE: (): TableNode | undefined => {
+                    return createTableNode({});
+                },
+                TR: (): TableRowNode | undefined => {
+                    return createTableRowNode({});
+                },
```

Comment:
Now if you will paste table with rowspan/colspan it will work, you will not be able to decrease the value. I think it is not a blocker for the initial release and then decide what should we do, either add controls for row/column spans, or forbid such behaviour

Reply:
> Now if you will paste table with rowspan/colspan it will work

Will it preserve the `colspan` attribute in the scenario described above? Are you sure?

Reply:
yes it should, we have colspan and rowspan in the model